run.py

```diff
@@ -14,19 +12,21 @@
 SKIP_DIRECTORY = "skip"
 COMMON_SKIP_FILE = "common.txt"
 LOG_FILE_PATH = os.path.join(LOG_DIRECTORY, LOG_FILENAME)
-TEST_DIRECTORY = os.path.join(os.getcwd(), "mongo/jstests")
 COMMON_LIST = os.path.join(SKIP_DIRECTORY, COMMON_SKIP_FILE)
+DOCKER_DIRECTORY = "/jstests"
 
 # User Variables (Consider moving these to a config file or environment variables)
 # Note: No username/password required for MaxScale test
-MONGO_HOST = "127.0.0.1"
-MONGO_PORT = 27017
-MONGO_USERNAME = "admin"
-MONGO_PASSWORD = "password"
+MONGO_HOST = os.environ.get("MONGO_HOST", "host.docker.internal")
+MONGO_PORT = int(os.environ.get("MONGO_PORT", 27017))
+MONGO_USERNAME = os.environ.get("MONGO_USERNAME", "admin")
+MONGO_PASSWORD = os.environ.get("MONGO_PASSWORD", "password")
 AUTHENTICATION_DATABASE = "admin"
 AUTHENTICATION_MECHANISM = "SCRAM-SHA-1"
 USE_SSL = "false"
 LOAD_BALANCE = "false"
+TEST_DIRECTORY = os.environ.get("TEST_DIRECTORY", os.path.join(os.getcwd(), "mongo/jstests"))
+DOCKER_COMMAND = f"docker compose exec legacy-mongo mongo"
```

Comment:
That requires `export COMPOSE_FILE=mongo5.yml`

Reply:
Also it requires the container to be running, which I can do `docker compose up` in dance